studio/components/interfaces/TableGridEditor/TableGridEditor.tsx

```diff
@@ -411,9 +411,8 @@ const TableGridEditor = ({
         {(isViewSelected || isTableSelected) && <TableDefinition id={selectedTable?.id} />}
       </SupabaseGrid>
 
-      {!isUndefined(selectedSchema) && (
+      {!isUndefined(snap.selectedSchemaName) && (
```

Comment:
lets just change this to `snap.selectedSchemaName !== undefined` and remove the unnecessary lodash dependency